e2e/run.sh

```diff
@@ -44,11 +44,8 @@ kubectl wait --for=condition=ready --timeout=300s pod -n thundernetes-system -l
 
 echo "-----Running end to end tests-----"
 cd cmd/e2e
-go build && ./e2e ${IMAGE_NAME_NETCORE_SAMPLE}:${IMAGE_TAG} && rm e2e
-
-echo "-----Running GameServer API tests-----"
-# create the gameserverapi namespace for the GameServer API tests
+# create the test namespaces
 kubectl create namespace gameserverapi
-go get github.com/onsi/ginkgo/v2
+kubectl create namespace mynamespace
```

Comment:
Nit: consider a better name ... this feels like a copy/paste of an example

Reply:
changed, thanks!